inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/pojo/source/SourceRequest.java

```diff
@@ -68,4 +68,6 @@
     @ApiModelProperty("Heartbeat of the source task")
     private String heartbeat;
 
+    @ApiModelProperty("Status")
```

Comment:
It's not recommended to add a status field in the request.

Operations that modify status should be encapsulated into a separate interface.